indexing-service/src/main/java/org/apache/druid/indexing/common/tasklogs/ConsoleLoggingEnforcementConfigurationFactory.java

```diff
@@ -143,6 +153,7 @@ private void applyConsoleAppender(LoggerConfig logger, Appender consoleAppender)
         // use the first appender's definition
         level = appenderRef.getLevel();
         filter = appenderRef.getFilter();
+        log.warn("clearing all configured loggers and adding console logger.");
```

Comment:
This should say it is `clearing all appenders for the logger {logger.getName()} and using console logger instead.`